Cpp/source/DrawingEngine/DrawingEngineTerminal.cpp

```diff
@@ -19,6 +52,26 @@ void DrawingEngineTerminal::show() const
 
 std::ostream& operator<<(std::ostream& os, const Pixel& obj)
 {
-    os << static_cast<int>(obj.R) << "," << static_cast<int>(obj.G) << "," << static_cast<int>(obj.B);
+    if(obj == Pixel{0,0,0})
+    {
+        os << ". ";
+    }
+    else if(obj == Pixel{10,10,10})
+    {
+        os << "* ";
+    }
+    else if(obj == Pixel{127,127,127})
+    {
+        os << "# ";
+    }
+    else
+    {
+        os << static_cast<int>(obj.R) << "," << static_cast<int>(obj.G) << "," << static_cast<int>(obj.B);
```

Comment:
it will break the shape of shown framebuffer

Reply:
fixed